Unix/configure

```diff
@@ -2298,6 +2300,12 @@ then
     cp $rootrelative/etc/omigen.conf \$destdir/$sysconfdir/omigen.conf
 fi
 
+
+if test ! -f "\$destdir/$sysconfdir/omireg.conf"
```

Comment:
This is the same syntax used for omiserver.conf, omicli.conf, and omigen.conf.  It was a cut-n-paste.  This is how it is usually done.